src/SML.py

```diff
@@ -1083,8 +1082,7 @@ def __init__(self, type: str, x: np.ndarray = None,
     if type == 'train':
       self.yt = DataSet(name="y", nrows=1, ncols=1)
       if yfile is not None and x is None:
-        y = np.loadtxt(yfile, skiprows=1, delimiter=",")
```

Comment:
Same issue as with empty `self.x.points`

Reply:
Yes, same as the previous comment:
temp = copy.deepcopy(np.loadtxt(
            yfile, skiprows=1, delimiter=","))
self.yt.points = temp.reshape(temp.shape[0], -1)